common/persistence/cassandra/metadata_store.go

```diff
@@ -212,7 +212,7 @@ func (m *MetadataStore) CreateNamespaceInV2Table(
 				}
 			}
 
-			msg := fmt.Sprintf("Namespace already exists.  NamespaceId: %v", existingID)
+			msg := fmt.Sprintf("Namespace already exists. Name: %q, NamespaceId: %v", request.Name, existingID)
```

Comment:
I end up avoiding `%q` and `"` in error messages because when they go to the logs, it looks ugly. Especially for strings that guaranteed not to have spaces in it (like namespace name). But we don't have well defined agreement on it so I am leaving it up to you.

Reply:
Btw, double space in the message means it was authored by @samarabbas :-)